proposals/nixpkgs-security.md

```diff
@@ -0,0 +1,278 @@
+# Sovereign Tech Fund / Nixpkgs Security Story
+
+## What open source infrastructure component would you like to contribute to? Please provide more details.
+
+[Nix](https://nix.dev/) is an open source build system, configuration management system, and mechanism for deploying software, focused on reproducibility. It is the basis of an ecosystem of exceptionally powerful tools – including Nixpkgs, [the largest, most up-to-date software repository in the world](https://repology.org/repositories/graphs), and NixOS, a Linux distribution that can be configured fully declaratively, with unmatched flexibility. For its proponents, Nix embodies technology sovereignty. It is leveraged and relied upon by computer enthusiasts, professionals, and organisations who recognise its value for all steps in the software development lifecycle.
+
+This project’s goal is to systematically leverage the very core of the Nix ecosystem: the Nix store. The store implements the [purely functional software deployment model](https://edolstra.github.io/pubs/phd-thesis.pdf): It deals with files in a file system like a modern computer program deals with objects in memory, by ensuring [referential integrity](https://en.m.wikipedia.org/wiki/Referential_integrity), and enabling garbage collection and transparent caching. It allows handling software builds like pure (that is, side-effect-free) computations in a declarative programming 
```

Comment:
I think this paragraph doesn't really convey that this application is about secure booting in NixOS, and NixOS as a project is also not mentioned here. The current form seems too generic.

Reply:
```suggestion
```

Reply:
OK, will modify a bit